app/bundles/FormBundle/Model/SubmissionModel.php

```diff
@@ -1130,4 +1130,89 @@ protected function validateActionCallbacks(SubmissionEvent $event, &$validationE
 
         unset($args, $actions, $availableActions);
     }
+
+    /**
+     * Return Forms with results to contacts tab
+     *ľ.
+     *
+     * @param null $leadId
+     * @param bool $inContactTab
+     *
+     * @return array
+     */
+    public function getFormsWithResults($leadId = null, $inContactTab = false)
+    {
+        $formResults    = [];
+        $viewOnlyFields = $this->formModel->getCustomComponents()['viewOnlyFields'];
+
+        $filters = [];
+
+        if ($inContactTab) {
+            $filters[] = ['column' => 'f.inContactTab', 'expr' => 'eq', 'value' => 1];
+        }
+
+        $permissions = $this->security->isGranted(
+            ['form:forms:viewown', 'form:forms:viewother'],
+            'RETURN_ARRAY'
+        );
+        if ($permissions['form:forms:viewown'] || $permissions['form:forms:viewother']) {
+            if (!$permissions['form:forms:viewother']) {
```

Comment:
Is the first if necessary here?

Reply:
It's from  here. I am gonna fixed it https://github.com/mautic/mautic/blob/afb4da4491ac2f12e8500d504906bd20f8ff47ee/app/bundles/FormBundle/EventListener/SearchSubscriber.php#L71